api/DOMRectReadOnly.json

```diff
@@ -65,16 +65,16 @@
           "description": "<code>DOMRectReadOnly()</code> constructor",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "61"
             },
             "chrome_android": {
-              "version_added": true
+              "version_added": "61"
             },
             "edge": {
               "version_added": "79"
             },
             "firefox": {
-              "version_added": true
+              "version_added": "62"
```

Comment:
This changes Firefox but isn't mentioned in the description. Was this a mistake?

Reply:
Yeah, that was a mistake, probably during the cherry-picking from the collector.  Reverting!